### ISSUE TITLE
Check Max Upload Size on Install

### ISSUE THREAD

**Issue by ArchBlood** [humhub/humhub#4904]
### Is your feature request related to a problem? Please describe.
At the moment on install the installer doesn't tell the `upload_max_filesize` in `SelfTest.php`

### Describe the solution you'd like
Add the following to `SelfTest.php`
```php
        // Checks Max Upload Size Extension
        $title = 'PHP - Max Upload Size';

        if (ini_get('upload_max_filesize')) {
            $checks[] = [
                'title' => Yii::t('base', $title),
                'state' => 'OK',
                'hint' => 'PHP reported a maximum of (upload_max_filesize): ' . ini_get('upload_max_filesize')
            ];
        }
```
![Screenshot_2](https://user-images.githubusercontent.com/35392110/108640124-e8221a00-7465-11eb-8b44-f7e03c42f72e.png)

### Describe alternatives you've considered
Not sure of a better solution for this, I'm already aware of the setting in `/admin/setting/file` but this is pretty vague but isn't shown on install so a developer can't really do much till they look through the settings

**Comment by luke-** [comment]
Could this also be a solution?
https://github.com/humhub/humhub/issues/4905

The maximum upload size can be set very low and there should be no problems with the use of HumHub?

**Comment by ArchBlood** [comment]
> Could this also be a solution?
> #4905
> 
> The maximum upload size can be set very low and there should be no problems with the use of HumHub?

This would probably work, although I'm not sure how this would be implemented on install? The above code provided shows the current max file size as shown in the screenshot, this also shows on the installer page as well, I'm just worried that redirecting the user will confuse or may even hinder the install process :thinking:

**Comment by luke-** [comment]
@ArchBlood Your example code snippet shows the current max upload size and doesn't implement any check. Does this have to be done in the installer?

**Comment by ArchBlood** [comment]
> @ArchBlood Your example code snippet shows the current max upload size and doesn't implement any check. Does this have to be done in the installer?

That is true to an extent, this code only checks the current `max_upload_size` and displays it, as for implementing on installer I believe it would help a lot of users, see the below post from the community site

https://community.humhub.com/s/general-discussion/?contentId=237058

**Comment by luke-** [comment]
Yes, probably the PHPInfo would be good here as well, here also the path to the used configuration file would be displayed directly.
